src/custom-element.js

```diff
@@ -694,6 +694,10 @@ function createBaseCustomElementClass(win) {
         );
       }
 
+      if (this.hasAttribute('disable-inline-width')) {
```

Comment:
This blocks `heights` behavior. Additionally, we'd need to set `sizeList_ = null`. I think this should just be handled as part of the ternary in the sizes block below.